acme.go

```diff
@@ -269,6 +272,77 @@ func pollAuthz(client *http.Client, url string) (*Authorization, error) {
 	return &auth, nil
 }
 
+// newCert requests a new certificate.
+//
+// If client argument is nil, DefaultClient will be used.
+// notBefore and notAfter are optional (if set to zero value)
```

Comment:
Remove `(if set to zero value)`. It seems redundant. I mean, if not set to zero, what does it make them, not optional? :)